Commiting AWS Batch Lab (#132)
aws-samples/aws-hpc-tutorials: content/05-AWS Batch/07-setup-batch-job-definition.md

tags = ["tutorial", "install", "AWS", "batch", "packer"]
+++

> In this step, you will set up a template for your jobs, known as a job definition. A job definition is not required, but is good practice so that you can keep track of how your jobs are launched. For more information about job definitions, see [Job Definitions](https://docs.aws.amazon.com/batch/latest/userguide/job_definitions.html).

Reviewer: ```suggestion
In this step, you will set up a template for your jobs, known as a **Job Definition (JD)**. For more information about **Job Definitions**, see [Job Definitions](https://docs.aws.amazon.com/batch/latest/userguide/job_definitions.html).
```
